app/models/wyeworker.rb

```diff
@@ -4,12 +4,15 @@
 #
 #  id         :integer          not null, primary key
 #  name       :string           not null
+#  type       :string
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
 #
 class Wyeworker < ApplicationRecord
   has_many :wyeworker_initiative_belongings
   has_many :initiatives, through: :wyeworker_initiative_belongings
 
-  validates :name, presence: true
+  before_create do
+    self.name = self.name || "Anonymous wyeworker"
+  end
```

Comment:
On an inheritance if a callback is declared on the parent then the child will inherit the callback so the doble definition in the parent and child is not necessary

Reply:
I changed "wyeworker" to "manager" in the child. But in any case, I will remove these callbacks, to demand that a name is provided